content/design-systems-for-developers/react/en/distribute.md

```diff
@@ -303,20 +303,28 @@ yarn add <your-username>-learnstorybook-design-system
 Now, let’s update the example app’s `.storybook/config.js` to list the design system components, and to use the global styles defined by the design system. Edit `.storybook/config.js` to:
 
 ```javascript
-import React from ‘react’;
-import { configure. addDecorator } from '@storybook/react';
-import { GlobalStyles } from ‘<your-username>-learnstorybook-design-system’;
-addDecorator(s => <><GlobalStyles/>{s()}</>);
+import React from "react";
+import { configure, addDecorator } from "@storybook/react";
+import { global } from "<your-username>-learnstorybook-design-system";
```

Comment:
I recently made a change on this repo & in several of the consumer apps to avoid importing `global` like this in the storybook config file:

https://github.com/chromaui/learnstorybook.com/blob/b6cd9ef72daedce5173e37d64781171dee7ea5d6/.storybook/config.js#L6

It conflicts with the another `global`:

https://github.com/chromaui/learnstorybook.com/blob/b6cd9ef72daedce5173e37d64781171dee7ea5d6/.storybook/config.js#L31

We might want to rethink that at some point as it might affect certain users and lead to confusion.

cc @tmeasday

Reply:
Right, it was probably a mistake to call the export `global` in the first place. Let's change that in the design system and here eventually.